script/openqa-clone-job

```diff
@@ -21,111 +21,114 @@
 
 =head1 openqa-clone-job
 
-openqa-clone-job - clone job from local or remote openQA instances
+Clones a job from the local or another openQA instance. Downloads all assets associated
+with the job (unless --skip-download is specified). Optionally settings can be modified.
 
 =head1 SYNOPSIS
 
   openqa-clone-job [OPTIONS] JOBREF [KEY=[VALUE] ...]
 
+  # clones job 42 (and parents) from "openqa.opensuse.org" to the local openQA instance
   openqa-clone-job https://openqa.opensuse.org/t42
-
   openqa-clone-job --from https://openqa.opensuse.org/tests/42
-
   openqa-clone-job --from https://openqa.opensuse.org 42
 
-  openqa-clone-job --from https://openqa.opensuse.org --host openqa.example.com 42
+  # clones job 42 (and parents) from "openqa.opensuse.org" to the openQA instance "openqa.example.com"
+  openqa-clone-job --skip-download --from https://openqa.opensuse.org --host openqa.example.com 42
+
+  # clones job 42 (and parents) within the local openQA instance modifying some job settings
+  openqa-clone-job --within-instance 42 MAKETESTSNAPSHOTS=1 FOOBAR=
+
+  # clones job 42 including its children but excluding its chained parents (e.g. to clone a
+  # full parallel cluster if job 42 is a parallel parent like a "server job")
+  openqa-clone-job --skip-chained-deps --clone-children https://openqa.opensuse.org/tests/42
+
+=head1 DESCRIPTION
+
+Call with either a full URL pointing to a test job to clone from or one of
+both parameters C<--from> or C<--within-instance>. The job ID can be specified
+as part of the URL or as its own parameter.
+
+Any parent jobs (chained or parallel) are also cloned unless C<--skip-deps> or
+C<--skip-chained-deps> is specified. If C<--skip-chained-deps> is specified
+published assets generated by parent jobs are downloaded to be directly used
+instead of generated. Keep in mind that by default any additionally specified job settinngns
```

Comment:
s/settinngns/settings/

and it looks like you haven't consistently wrapped at 80 characters. I suggest you find a way how to auto-wrap in your editor of choice as we do not have automatic checks for that. As this text is shown to the user we should be consistent there

Reply:
I can make the wrapping consistent within the code but I'd like to note that the Perl module we use to display this as `--help`-text applies its own wrapping so this is not actually shown to the user.